odoo/http.py

```diff
@@ -560,6 +560,21 @@ def serialize_exception(exception, *, message=None, arguments=None):
     }
 
 
+def serialize_exception_like_json2(controller_endpoint):
```

Comment:
I think we should not have something like this in core. `mail` can patch the response as needed.